src/slack-send.js

```diff
@@ -114,6 +116,10 @@ module.exports = async function slackSend(core) {
 
     const time = (new Date()).toTimeString();
     core.setOutput('time', time);
+
+    if (webResponse && webResponse.ts !== undefined) {
+      core.setOutput('message-ts', webResponse.ts);
```

Comment:
If you return thread-ts instead, we can check if there is thread_ts first and then fall back to message.ts. Also, it may be fine to return both message-ts and thread-ts.